question/bank/previewquestion/lang/en/qbank_previewquestion.php

```diff
@@ -23,5 +23,10 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+$string['comments'] = 'Comments';
 $string['pluginname'] = 'Preview question bank feature';
+$string['previewoptions'] = 'Preview options';
 $string['privacy:metadata'] = 'Preview question plugin shows a question in a pop-up, it does not store any user data.';
+$string['questionversion'] = 'Question version';
```

Comment:
please differentiate the strings using comment, like for which part this string is for

Reply:
fixed